Product/Production/Adapters/PatientDiscovery_a0/src/main/java/gov/hhs/fha/nhinc/patientlocationquery/adapter/AdapterPatientLocationQueryUnsecured.java

```diff
@@ -47,7 +48,8 @@
     public AdapterPatientLocationQueryResponseType adapterPatientLocationQuery(
         AdapterPatientLocationQueryRequestType adapterPatientLocationQueryRequest) {
 
-        return new AdapterPatientLocationQueryResponseType();
+        AdapterPatientLocationQueryRequestType msg = new AdapterPatientLocationQueryRequestType();
```

Comment:
Why are we creating a new request, when we should be pulling it from the endpoint call?